.azure-pipelines/linux-CI-nightly.yml

```diff
@@ -15,28 +15,22 @@ jobs:
     matrix:
       Py37-Onnx160-Sk0221:
         python.version: '3.7'
-        numpy.version: '==1.18.0'
+        numpy.version: '==1.18.1'
```

Comment:
skl2onnx can only work with this latest numpy release?

Reply:
No both. I usually choose the last one of series (1.17.5, 1.18.1).